openfl/interface/pki.py

```diff
@@ -144,20 +125,12 @@ def get_token_(name, ca_url, ca_path):
     "-p",
     "--ca-path",
     default=".",
-    help="The path to CA client",
+    help="Path to CA client, defaults to current directory.",
```

Comment:
I saw in other commands the pattern was `show_default=True`, and not explaining the default in the help message. Is there a reason not to be consistent?

Reply:
I applied the following logic:
* If `default` value is comprehensible, help string need not describe it - as it may change and a developer/reviewer may miss updating the `help` string.
* If `default` is not comprehensible, in this case `.` is being shorthand for current directory.